### DISCUSSION TITLE
WDIO Image Comparison question

### DISCUSSION THREAD

**Discussion by VladGoogle** [webdriverio/webdriverio#9671] [Q&A] [answered]
I want to take a full-page screenshot of a dashboard page using the Image Comparison service, and this is what it looks like (see attachment)
I have a question: is there any way to remove these red lines from the screenshot, or just save the screenshot of the page without scrolling the page itself?
![Dashboard](https://user-images.githubusercontent.com/71459589/214837531-54aa7b43-fa44-4a7c-8003-0ef7143befea.png)

**Comment by christian-bromann** [comment]
I recommend to file an issue in the [project repository](https://github.com/wswebcreation/wdio-image-comparison-service).

**Comment by wswebcreation** [comment, marked as answer]
I assume the red part is an element that can be hidden, then you can use `hideElements` as an option, see https://github.com/wswebcreation/webdriver-image-comparison/blob/main/docs/OPTIONS.md#hideelements for the details. 

If you are using typescript you will see the implementation, but this is an example https://github.com/wswebcreation/wdio-image-comparison-service/blob/e622723212aeb93b7b7d8dd86d6427bb8568ee2a/types/index.d.ts#L17

**Reply by VladGoogle** [reply]
Thanks a lot, it solved the problem

**Comment by umarkhanTR** [comment]
Can we do partial comparison of two images. The scenario is that our website is consumer facing and a rating dialog appears on the web page randomly, towards the bottom of the page. a Small dialog. I want to see if for every page if that dialog is displayed and if it is then click on the close button. Please advice.

**Reply by christian-bromann** [reply]
@umarkhanTR as mentioned above you can exclude certain elements which will be colored in black, see https://github.com/wswebcreation/webdriver-image-comparison/blob/main/docs/OPTIONS.md#hideelements
